CDS Maps/Individual Maps/CDS-to-FHIR/nNGM-CDS-to-FHIR-Systemische Therapie.map

```diff
@@ -1,20 +1,7 @@
 /// version = 0.1
 /// title = "nNGM_Mapping_SystemischeTherapieFHIR"
 
-/*
-    TODO
-    MedicationAdministration
-    - what to do with uuid_nngm_MedicationAdministration_changed?
-    - Missing extension.url for Anzahl der Zyklen (id_1250)
-        profile requires dosage.rate.value, .system and .code
-    - Missing valueSet for id_2342
-    - When 'Abbruch wegen Progress' option is checked in Art des Therapieendes (id_1356), it displays a field named 'Datum Progess' (blockindex = 6, groupindex = 0, itemid = 'sy_date_progress'),
-        but its information is not on Seatable nor Master excel file, so it's not being mapped
-    - id_1465 isn't mapping properly because it cannot override value already set by id_1246, even though it's effectivePeriod.start and id_1246 it's effective.start
-
-    EpisodeOfCare
-    - what to do with uuid_nngm_dateOfProgress (and/or uuid_nngm_dateOfProgress2)?
-*/
+// Todo: Extension(Date) needed for Datum Aenderung (id_1465)
```

Comment:
Is this TODO still relevant? If so, let's solve it first 👍

Reply:
not relevant anymore